src/components/widgets/WeatherWidget.tsx

```diff
@@ -30,8 +30,8 @@ export const WeatherWidget = ({ navigation }: { navigation: NavigationScreenProp
   useHomeRefresh(refetch);
 
   return (
-    <Touchable
-      onPress={() => navigation?.navigate('Weather', { weatherData: { icon, temperature } })}
+    <TouchableOpacity
+      onPress={() => navigation?.navigate('Weather', { title: text ?? texts.screenTitles.weather })}
```

Comment:
the other widgets have `useCallback` navigation onpresses, right? but here not?!

Reply:
i missed that back when i implemented it i guess. it is inside a useCallback before now.

i also fixed the dependencies of all the onpress callbacks of the widgets